packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.types.ts

```diff
@@ -46,6 +46,34 @@ export interface IScrollablePaneProps extends React.HTMLAttributes<HTMLElement |
   initialScrollPosition?: number;
 
   scrollbarVisibility?: ScrollbarVisibility;
+
+  stickyAboveContainerBehavior?: IStickyContainerBehavior;
+  stickyBelowContainerBehavior?: IStickyContainerBehavior;
+}
+
+export interface IStickyContainerBehavior {
+  /**
+   * If true, it replicates actual element instead of keeping placeholder
+   * Best Practices: Perf reasons
+   */
+  notUsePlaceHolder: boolean;
+  /**
+   * If true, arranges stickies based on 'order' prop
+   */
+  arrangeStickiesBasedOnOrder: boolean;
```

Comment:
What if the order 'prop' is not provided? I would rather assume that we rely on the order prop to identify if we should 'arrangeStickiesBasedOnOrder'.

Reply:
Let's say there are 3 sticky components (stickyPosition='Header')
1. what if only for one component 'order' prop is provided ? A fallback logic would be needed.
```
if (arrangeStickiesBasedOnOrder) {
      return (a.props.order || 0) > (b.props.order || 0);
    } 
```
2. It would be required to iterate through stickies[] to know if stickies are to be arranged based on prop.

Reply:
When @tharshada and I last spoke, sorting based on order was an all or nothing approach. The presence of `order` was an optimization that opted-in all of the sticky elements within a container.

I think that's acceptable, but understand that the interface may confuse folks who only provide the value for _some of_ the sticky elements.

Reply:
sticky components which 've same sticky position are a part of same container (stickyAbove container or stickyBelow container). It made me put the prop at the container level and not for individual sticky. If sticky component doesn't 've value for order prop, it is assumed to be 0.
https://github.com/OfficeDev/office-ui-fabric-react/pull/9060#discussion_r286051586